CMakeLists.txt

```diff
@@ -201,6 +223,14 @@ if (CONFIG_BUILD_WITH_TFM)
     set(TFM_PROFILE_ARG BUILD_PROFILE ${CONFIG_TFM_PROFILE})
   endif()
 
+  # Enable TFM partitions as specified in Kconfig
+  set(TFM_ENABLED_PARTITIONS_ARG ENABLED_PARTITIONS)
```

Comment:
I would remove this line, and place the argument name `ENABLED_PARTITIONS` in the call for better readability.

It is ok for the following list of partitions to be empty (and `list(APPEND)` will create the list if not yet present ).
```suggestion
```